spec/unit/blockscore/question_set_spec.rb

```diff
@@ -38,17 +38,30 @@ module BlockScore
       let(:incorrect_answers) { QuestionSetHelper.incorrect_answers(question_set.questions) }
 
       context 'correct answers' do
-        before { question_set.score(correct_answers) }
-
-        its(:score) { should eq 100.0 }
-        it { expect(question_set.score(correct_answers)).to eq 100.0 }
+        it { expect(question_set.score(correct_answers)).to be 100.0 }
```

Comment:
I'd generally put the argument to rspec matchers in parentheses. Reading "2 whitespace in a row" always puts doubt in the mind of the reader, since the ruby syntax easily "bites" people in terms of association.

I realize the reminder of the code base might not follow my suggestion right now, so only changing the instances in this diff would increase entropy.

Just placing this note here to start a discussion potentially resulting in a global change, or rspec specific rubocop configuration to enforce a consistent style.